charts/clamav/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.podDisruptionBudget -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "clamav.fullname" . }}
```

Comment:
Could you add the default labels (see the other resources) ?

Reply:
yep.. will do all related changes.